pgh_play/www/css/index.css

```diff
@@ -2,6 +2,7 @@
 	height: 660;
```

Comment:
Instead of absolute values, perhaps have both height/width fill container and rely on relative values of child elements to automatically adjust to the screen size.